Facebook.AudienceNetwork.iOS/build.cake

```diff
@@ -1,10 +1,10 @@
 
 #load "../common.cake"
 
-var SDK_VERSION = "4.24.0";
-var SDK_URL = string.Format ("https://origincache.facebook.com/developers/resources/?id=FacebookSDKs-iOS-{0}.zip", SDK_VERSION);
-var SDK_FILE = "FacebookSDKs-iOS.zip";
-var SDK_PATH = "./externals/FacebookSDKs";
+var SDK_VERSION = "4.25.0";
+var SDK_URL = string.Format ("https://origincache.facebook.com/developers/resources/?id=FBAudienceNetwork-{0}.zip", SDK_VERSION);
```

Comment:
Looks good, but I just wanted to know why the package name changed? Was it originally a bulk package with everything and they split it?